scss/_functions.scss

```diff
@@ -137,7 +137,7 @@ $_luminance-list: .0008 .001 .0011 .0013 .0015 .0017 .002 .0022 .0025 .0027 .003
   );
 
   @each $name, $value in $rgb {
-    $value: if($value / 255 < .03928, $value / 255 / 12.92, nth($_luminance-list, $value + 1));
+    $value: if($value * .555 < .03928, divide($value * .555, 12.92), nth($_luminance-list, $value + 1));
     $rgb: map-merge($rgb, ($name: $value));
```

Comment:
Same issue as above. In addition, the divide() function could be removed completely (1/255/12.92) = * .0003035

Reply:
I think moving away from 255 - a recognizable number with obvious context when dealing with RGB - to some magic numbers is a bad idea. I'd argue that the reason this dividing twice wasn't previously simplified was for the same reason: to keep the 255. If we have a divide function, we should use it and preserve the readability and ease of maintenance.

Reply:
Or a little comment should be added that states : 0.0003035 = 1/255/12.92

Reply:
And to be honest, maybe the .03928 and 12.92 values also deserve a comment ;)